### ISSUE TITLE
Running a select on all measurements with regexp gives error

### ISSUE THREAD

**Issue by mjdesa** [influxdb/influxdb#3128]
On OSX with influx version 0.9 built from source

running the query

```
SELECT * FROM /.*/ LIMIT 1
```

yields a

```
ERR: can not use field in group by clause: path
```

in return.

@dgnorton, @otoolep

**Comment by beckettsean** [comment]
@otoolep given that this is from master, not 0.9.0, I wonder if this is an unintended side effect of the `select *` changes you made?

**Comment by otoolep** [comment]
Hmmmm, perhaps. I'll take a look.

**Comment by otoolep** [comment]
No issue on Linux, with code at 9b964f93cc8cf6914eb1c5a2ca8a044f393b419d.

```
> create database foo
> use foo
Using database foo
> select * from /.*/ limit 1
> 
```

Please re-open if you can reproduce.

**Comment by jwilder** [comment]
Fixed by #3616

**Comment by otoolep** [comment]
Good man @jwilder
